recipe/meta.yaml

```
@@ -42,7 +42,8 @@ requirements:
     - menuinst >=2
     - packaging
     - patch >=2.6                  # [not win]
-    - patchelf                     # [linux]
+    # cap due to https://github.com/conda/conda-build/issues/4881; https://github.com/NixOS/patchelf/issues/492
```

Suggesting a little more description and dropping links below for readability

```suggestion
    # Cap to avoid bug where ELF load command is garbled.
    # 
    # xref: https://github.com/conda/conda-build/issues/4881
    # xref: https://github.com/NixOS/patchelf/issues/492
```